docs/getting-started/architecture-and-components/registry.md

```diff
@@ -1,15 +1,60 @@
 # Registry
 
-The Feast feature registry is a central catalog of all the feature definitions and their related metadata. It allows data scientists to search, discover, and collaborate on new features.
+Feast uses a registry to store all applied Feast objects (e.g. Feature views, entities, etc). It allows data scientists to search, discover, and collaborate on new features. The registry exposes methods to apply, list, retrieve and delete these objects, and is an abstraction with multiple implementations.
 
-Each Feast deployment has a single feature registry. Feast only supports file-based registries today, but supports four different backends.
+Feast comes with built-in file-based and sql-based registry implementations. By default, Feast uses a file-based registry, which stores the protobuf representation of the registry as a serialized file in the local file system. For more details on which registries are supported, please see [Registries](../../reference/registries/).
```

Comment:
I'm not sure that path will work 🤔

Reply:
Not sure, I mimicked this [line](https://github.com/feast-dev/feast/blob/master/docs/getting-started/architecture-and-components/offline-store.md?plain=1#L5) from offline store markdown.

Reply:
Oh yes it will nvm.